htdocs/core/class/commonobject.class.php

```diff
@@ -4331,12 +4332,20 @@ public function fetchObjectLinked($sourceid = null, $sourcetype = '', $targetid
 					$classPath = $element_properties['classpath'];
 					$classFile = $element_properties['classfile'];
 					$className = $element_properties['classname'];
-					$module = $element_properties['module'];
+					if ($objecttype == 'subscription') {
```

Comment:
We should not have exception here. Exception must be managed at one place: inside the method getElementProperties() executed 3 lines before.

Reply:
thanks, I'll take a look at that